internal/namespaces/rdb/v1/custom_backup.go

```diff
@@ -112,3 +118,97 @@ func backupRestoreBuilder(c *core.Command) *core.Command {
 
 	return c
 }
+
+func getDefaultFileName(rawURL string) (string, error) {
+	u, err := url.Parse(rawURL)
+	if err != nil {
+		return "", err
+	}
+	splitURL := strings.Split(u.Path, "/")
+	filename := splitURL[len(splitURL)-1]
+	return filename, nil
+}
+
+func backupDownloadCommand() *core.Command {
+	type backupDownloadArgs struct {
+		BackupID string
+		Region   scw.Region
+		Output   string
+	}
+
+	return &core.Command{
+		Short:     `Download a backup locally`,
+		Long:      `Download a backup locally.`,
+		Namespace: "rdb",
+		Resource:  "backup",
+		Verb:      "download",
+		ArgsType:  reflect.TypeOf(backupDownloadArgs{}),
+		Run: func(ctx context.Context, argsI interface{}) (i interface{}, err error) {
+			args := argsI.(*backupDownloadArgs)
+			api := rdb.NewAPI(core.ExtractClient(ctx))
+			backup, err := api.WaitForDatabaseBackup(&rdb.WaitForDatabaseBackupRequest{
+				DatabaseBackupID: args.BackupID,
+				Region:           args.Region,
+				Timeout:          scw.TimeDurationPtr(backupActionTimeout),
+				RetryInterval:    core.DefaultRetryInterval,
+			})
+			if err != nil {
+				return nil, err
+			}
+			if backup.DownloadURL == nil {
+				return nil, fmt.Errorf("no download URL found")
+			}
+
+			res, err := http.Get(*backup.DownloadURL)
+			if err != nil {
+				return nil, err
+			}
+			defer res.Body.Close()
+
+			// Create the file
+			filename, err := getDefaultFileName(*backup.DownloadURL)
+			if err != nil {
+				return nil, err
+			}
+			if args.Output != "" {
+				filename = args.Output
+			}
+
+			out, err := os.Create(filename)
+			if err != nil {
+				return nil, err
+			}
+			defer out.Close()
+
+			// Write the body to file
+			size, err := io.Copy(out, res.Body)
+			if err != nil {
+				return nil, err
+			}
+			sizeStr, err := human.Marshal(scw.Size(size), nil)
+			if err != nil {
+				return nil, err
+			}
+			return fmt.Sprintf("Backup downloaded to %s successfully (%s written)", filename, sizeStr), nil
```

Comment:
This type of return is only design for human marshaling. It would be better to have a custom `type backupResult` that can be serialized in JSON correctly and have a custom human marshal func